machines/store.ts

```diff
@@ -176,6 +211,19 @@ export const storeMachine =
 
       services: {
         clear,
+        checkStorageInitialisedOrNot: () => async (callback) => {
+          const isDirectoryExist = await Storage.isVCStorageInitialised();
+          // if dir exists --> Can have data --> show popUp
+          // else --> clear data
+          if (!isDirectoryExist) {
+            // clear the storage
+            callback(model.events.READY());
+          } else {
+            // ideally sendToParent()
+            callback(model.events.ERROR(new Error('show the popup')));
```

Comment:
Should the error message here be like "Error in storage......" rather than "Show the popup"?
& can this be logged too?